### ISSUE TITLE
[Bug]:Getting 'UNABLE_TO_GET_ISSUER_CERT_LOCALLY' while installing playwright for my javascript project. we are using Zscaler.

### ISSUE THREAD

**Issue by abhijitgithub2019** [microsoft/playwright#32412]
### Version

latest

### Steps to reproduce

npm init playwright@latest

<img width="1728" alt="Screenshot 2024-09-02 at 8 45 20 PM" src="https://github.com/user-attachments/assets/07277151-cdce-411a-9a7d-8eb053f78377">

### Expected behavior

Will get the code: 'UNABLE_TO_GET_ISSUER_CERT_LOCALLY' error.
<img width="707" alt="playwright_issue" src="https://github.com/user-attachments/assets/67414b70-65a2-4696-a119-cef5263ff22d">

### Actual behavior

The install should be successfull.

### Environment

```shell
macOS: 14.5
node : v20.15.1
npm : 10.7.0
```

**Comment by tyler-suard-parker** [comment]
Same, but using windows

**Comment by abhijitgithub2019** [comment]
Yes, I am in a cooperate environment, Thanks, I have tried the above guide. I am getting
<img width="1652" alt="Screenshot 2024-09-03 at 5 28 11 PM" src="https://github.com/user-attachments/assets/14a6b326-5ee5-4608-8e6f-6e8c2e2fb07d">
 the below error.

**Comment by pavelfeldman** [comment]
You should refer to your network administrator and they will help you pick one of the workarounds for the networks behind the proxy from https://playwright.dev/docs/browsers#install-behind-a-firewall-or-a-proxy. It is unlikely that we can help from here w/o the details on your network config and this does not seem to be a Playwright bug.

**Comment by rajsite** [comment]
Ran into similar errors:

> playwright Error: unable to get local issuer certificate

And setting env var `NODE_USE_SYSTEM_CA=1` in a recent node version helped for me on an enterprise managed system. See [Node docs: Adding CA Certificates from the System Store](https://nodejs.org/en/learn/http/enterprise-network-configuration#adding-ca-certificates-from-the-system-store).
